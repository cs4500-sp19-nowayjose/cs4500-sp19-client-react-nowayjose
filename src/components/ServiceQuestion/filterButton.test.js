import React from 'react';
import FilterButton from './filterButton';
import renderer from 'react-test-renderer';

test('Render filter button', function() {
  const component = renderer.create(
    <FilterButton
      onClickFilterData={() => {}}
    />
  )
  let tree = component.toJSON()
  return expect(tree).toMatchSnapshot()
})