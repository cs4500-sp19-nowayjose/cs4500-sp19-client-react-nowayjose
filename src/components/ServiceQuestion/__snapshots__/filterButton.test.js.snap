// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render filter button 1`] = `
<button
  className="btn-lg btn"
  onClick={[Function]}
  style={
    Object {
      "color": "#FEC107",
      "position": "fixed",
      "right": 10,
    }
  }
>
  <i
    className="fas fa-search fa-2x"
  />
</button>
`;
